src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigArgs.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+#nullable enable
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal partial class ConfigPathsArgs
```

Comment:
why partial?

Reply:
This code was originally autogenerated using a template, but now that we moved it to `NuGet.CommandLine.XPlat` we're no longer using the template tool to generate it; I'll remove the partial modifier.